packages/react-components/config/api-extractor.local.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "extends": "./api-extractor.json",
+  "mainEntryPointFilePath": "<projectFolder>/dist/<unscopedPackageName>/src/index.d.ts",
+  "compiler": {
+    /* This is a quickfix to make build:local work - @see https://github.com/microsoft/fluentui/issues/19360 */
```

Comment:
temporary fix just for react-components as the issue didnt surface in other vNext packages yet https://github.com/microsoft/fluentui/issues/19360

Reply:
Could you add a comment about which config this is copied from/meant to mirror?

Reply:
it's the tsconfig present in every package (same logic as in api-extractor just task), but good point to explicitly point anyone to right direction/reasoning